lib/twitter_ads.rb

```diff
@@ -7,9 +7,11 @@
 #
 require 'oauth'
 require 'json'
+require 'multi_json'
+
+ADS_API_ENDPOINT = 'ads-api.twitter.com'
+TRACE = false
```

Comment:
Good question, one thing I suggest is place those constants inside the module TwitterAds so that they can be accessed via TwitterAds::ADS_API_ENDPOINT and TwitterAds::TRACE.